client.go

```diff
@@ -747,14 +756,19 @@ func (c *Client) connect(maxRetries int, node *Node) error {
 
 		var err error
 		if node == nil {
-			node, err = GetWsAddr(c.Address(), c.config)
+			c.peer = NewPeer(c.config.StunServerAddr.Elems())
```

Comment:
go sdk should use ws by default unless explicitly configured to use webrtc by user

Reply:
Yes, modified